Dockerfile

```diff
@@ -23,15 +48,28 @@ RUN echo "%sudo   ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers && \
 WORKDIR /home/steam
 USER steam
 
-ENV am_ark_SessionName="Ark Server" \
-    am_serverMap="TheIsland" \
-    am_ark_ServerAdminPassword="k3yb04rdc4t" \
+ENV am_ark_SessionName=Ark\ Server \
+    am_serverMap=TheIsland \
+    am_ark_ServerAdminPassword=k3yb04rdc4t \
     am_ark_MaxPlayers=70 \
     am_ark_QueryPort=27015 \
     am_ark_Port=7778 \
     am_ark_RCONPort=32330 \
-    am_arkwarnminutes=15
+    am_ark_AltSaveDirectoryName=SavedArks \
+    am_arkwarnminutes=15 \
+    am_arkAutoUpdateOnStart=false
+
+ENV VALIDATE_SAVE_EXISTS=false \
+    BACKUP_ONSTART=false \
+    LOG_RCONCHAT=0 \
+    ARKCLUSTER=false
 
+# only mount the steamapps directory
+VOLUME /home/steam/.steam/steamapps
 VOLUME /ark
+# optionally shared volumes between servers in a cluster
+VOLUME /arkserver
+# mount /arkserver/ShooterGame/Saved seperate for each server
```

Comment:
could you document the in the readme how to run properly 2 containers part of a cluster?

Reply:
I added an example that works out-of-the-box, I hope you can verify that.

Reply:
PS: until this is merged, you need to use `IMAGE=drpsychick/arkcluster`